compiler/resolution/lowerIterators.cpp

```diff
@@ -2106,7 +2173,8 @@ expandBodyForIteratorInline(ForLoop*       forLoop,
                             Symbol*        index,
                             bool           inTaskFn,
                             TaskFnCopyMap& taskFnCopies,
-                            bool&          addErrorArgToCall) {
+                            bool&          addErrorArgToCall,
+                            const SymbolMap& svarMap) {
```

Comment:
Either remove extra spaces (my choice) in all formals, or make sure to keep this like a table by adding more spaces above.